lib/internal/repl.js

```diff
@@ -59,7 +61,17 @@ function createRepl(env, cb) {
 }
 
 function setupHistory(repl, historyPath, ready) {
-  const fs = require('fs');
+  // default to using <homedir>/tmp/node_history_file
```

Comment:
Should just create `.node_history` in the user's home directory. I don't think it would be hidden though. I imagine it would be similar to `.npmrc` if npm does the same thing on Windows.